homeassistant/components/fritz/common.py

```diff
@@ -311,6 +311,20 @@ def unregister_entity_updates() -> None:
                 )
         return unregister_entity_updates
 
+    async def async_entity_states_update(self) -> dict:
+        """Run registered entity update calls."""
+
+        def _entity_states_update() -> dict:
```

Comment:
There's no need to nest this function. Make it a method instead on the class.

Reply:
did this before, but had the feeling that this looks more clean in the `_async_update_data()` ... but sure, if this doesn`t matter, than i'll remove the wrapping